### PR TITLE
Sync with Cirq 0.8

### PR DESCRIPTION
**Pull request by bryano**

### CODE REVIEW

File: openfermioncirq/gates/common_gates.py [hunk 1]
```diff
@@ -14,11 +14,10 @@
 
 from typing import Optional, Tuple
 
-import numpy as np
-
 import cirq
-import deprecation
+import numpy as np
 import sympy
+import deprecation
```
Comment (bryano) [inline openfermioncirq/gates/common_gates.py]: I wish I understood why pylint says this should be here.

File: openfermioncirq/trotter/algorithms/linear_swap_network.py [hunk 2]
```diff
@@ -115,16 +115,19 @@ def trotter_step(
...
 
+        if not isinstance(control_qubit, cirq.Qid):
+            raise TypeError('Control qudit must be specified.')
+
         # Apply one- and two-body interactions for half of the full time
         def one_and_two_body_interaction(p, q, a, b) -> cirq.OP_TREE:
             yield CRxxyy(
                     0.5 * self.hamiltonian.one_body[p, q].real * time).on(
-                            control_qubit, a, b)
+                            cast(cirq. Qid, control_qubit), a, b)
```
Comment (mpharrigan) [inline openfermioncirq/trotter/algorithms/linear_swap_network.py]: `cirq. Qid` -> `cirq.Qid`  [or is there something wrong with my font?]

Comment (bryano) [reply openfermioncirq/trotter/algorithms/linear_swap_network.py]: Done. How did that not get picked up by a test?!

Comment (mpharrigan) [reply openfermioncirq/trotter/algorithms/linear_swap_network.py]: Apparently it still works with the space there. But perhaps pylint should have complained

**Pull request merged**